public/js/components/FormFields/FormFieldTagPicker.js

```diff
@@ -39,20 +39,27 @@ export default class FormFieldTagPicker extends React.Component {
 
     const searchText = e.target.value;
 
-    this.setState({
-      searchText: searchText
-    });
-
-    searchTags(searchText, this.props.tagType).then((results) => {
+    if (searchText.length > 0) {
```

Comment:
Might be worth making this a minimum number of chars before the search starts. I've seen three used in other contexts but that'd depend on whether we have any one or two character tags I guess (but hope that's not the case)

Reply:
Great point! Is Composer doing that too? I'll have a look

Reply:
It kicks off with one character in Composer, so I think it's best to align AW with it. Though I think you're right 😞